Docker.md

```diff
@@ -1,20 +1,37 @@
-## development docker-compose.yaml file
+# Docker guidelines
 
-### Publishing ports
+## `Dockerfile`
+
+When building an image that is running some services,
+use the [`EXPOSE` directive](https://docs.docker.com/engine/reference/builder/#expose),
```

Comment:
`docker run` also has the `--expose` flag that does the same thing. This could help us in ListenBrainz where we use the same image for many tasks, but only expose a webserver on the container used for the actual LB website. This way, atj's script will correctly pick up that the webserver has an exposed port, but won't show false positives for other containers, Should we add this option to these guides?

Reply:
Or even replace the builder `EXPOSE` directive with the runner `--expose` flag?

Reply:
I added a proposed change to explain when to use EXPOSE or --expose